tests/integration/create-switch-account.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserDriver, createTestSelector, setupBrowser } from './utils';
+import { WalletPage } from './page-objects/wallet.page';
+import { ScreenPaths } from '@common/types';
+import { SettingsSelectors } from './settings.selectors';
+
+jest.setTimeout(40_000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+describe.only(`Create and switch account`, () => {
+  let browser: BrowserDriver;
+  let wallet: WalletPage;
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    wallet = await WalletPage.init(browser, ScreenPaths.INSTALLED);
+  }, 10000);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {}
+  });
+
+  it('should be able to create a new account then switch', async () => {
+    await wallet.signUp();
+    await wallet.clickSettingsButton();
+    await wallet.page.click(createTestSelector('settings-create-an-account'));
+    await wallet.page.waitForSelector(createTestSelector('create-account-done-button'))
+    await wallet.page.click(createTestSelector('create-account-done-button'));
+    const displayName = await wallet.page.textContent(
+      createTestSelector('home-current-display-name')
+    );
+    expect(displayName).toEqual('Account 2');
+    await wallet.clickSettingsButton();
+    await wallet.page.click(createTestSelector(SettingsSelectors.SwitchAccount));
+    await wallet.page.click(createTestSelector('switch-account-item-0'));
+    await wallet.page.waitForSelector(createTestSelector('account-checked-1'), {state: 'hidden'})
+    const displayName2 = await wallet.page.textContent(
+      createTestSelector('home-current-display-name')
+    );
+    expect(displayName2).toEqual('Account 1');
+  });
+
+  const COUNT = 3;
```

Comment:
i prefer consts to all be at the top of a page so they don't get lost

Reply:
Sure, I will do it.